settings.gradle

```diff
@@ -1,3 +1,5 @@
+include 'android'
+include 'inline'
```

Comment:
'inline' artifact, too? :)

What is the strategy for 'inline' artifact? I'd rather mockito worked out of the box for final classes (or have toggles in the API). I don't like the idea of separate mockito-inline artifact.